openslides/agenda/static/js/agenda/base.js

```diff
@@ -288,8 +288,6 @@ angular.module('OpenSlidesApp.agenda', ['OpenSlidesApp.users'])
     }
 ])
 
-// TODO: Remove all find() calls from the projector logic. It is also used on the site so this has to be
```

Comment:
Hey, please don't delete the TODO withou fullfilling it. If this factory is not used by the projector any more, move it to site.js.